assets/src/edit-story/components/checklist/checks/storyHasAmpErrors.js

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import { useMemo } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { useStory } from '../../../app';
+import { ChecklistCard, DefaultFooterText } from '../../checklistCard';
+import { PRIORITY_COPY } from '../constants';
+import { useRegisterCheck } from '../countContext';
+
+export async function storyHasAmpErrors({ link, status }) {
```

Comment:
Maybe `getAmpValidationErrors`?

It's otherwise a bit confusing because the React component has the same name, just with PascalCase.

Reply:
totally get that, this is a pattern with all the other checks too - i suppose the thinking here was that both the function and the React component create the total `check`.   Open to changing it, especially if it's confusing to someone who hasn't been in that section of the code base as much as pea pod has.

Reply:
🤝  went with `getStoryAmpValidationErrors` and `StoryAmpValidationErrors`